public/pages/Indices/utils/constants.tsx

```diff
@@ -45,7 +46,7 @@ const getColumns = (props: IColumnOptions): EuiTableFieldDataColumnType<ManagedC
       sortable: true,
       truncateText: false,
       textOnly: true,
-      width: "250px",
+      width: "320px",
```

Comment:
Would it be possible to let user drag the width of our table?
Currently although the text is capped, I remember user can hover on it and see the name which is fine.

Adding width here is not the best because people may have very long index name 😅